static/app/views/replays/list/replaySearchBar.tsx

```diff
@@ -63,23 +69,70 @@ function getSupportedTags(supportedTags: TagCollection) {
   };
 }
 
+/**
+ * Returns 3 sections: replay fields, replay click fields, and (filtered, non-field) tags.
+ */
+function getFilterKeySections(
+  supportedTags: TagCollection,
+  organization: Organization
+): FilterKeySection[] {
+  if (!organization.features.includes('search-query-builder-replays')) {
+    return [];
+  }
+
+  const nonFieldTags: string[] = Object.values(supportedTags)
+    .map(tag => tag.key)
+    .filter(
+      key =>
+        !REPLAY_FIELDS.includes(key as ReplayFieldKey | FieldKey) &&
+        !REPLAY_CLICK_FIELDS.includes(key as ReplayClickFieldKey)
+    );
+  nonFieldTags.sort();
```

Comment:
Something which we did on the issues page is sort these tags by `totalValues`. That way the top tags you see are ones which are used the most.